src/scripts.js

```diff
@@ -147,3 +148,62 @@ function showTravelerInfo(traveler) {
   setTravelerTrips('present');
   setTravelerCostOverYear();
 };
+
+form.addEventListener('submit', showNewTrip)
+
+function getFormData(e) {
+  e.preventDefault();
+  const formData = new FormData(e.target);
+  const newTrip = {
+    id: tripsRepo.trips.length + 1,
+    userID: travelerId,
+    destinationID: (destinationRepo.destinations.find(destination => destination.destination === formData.get('destination-datalist'))).id,
+    travelers: formData.get('number-people-input'),
+    date: formData.get('departure-date-input'),
+    duration: formData.get('trip-length-input'),
+    status: 'pending',
+    suggestedActivities: []
+  };
+  e.target.reset();
+  resetTripsDisplay();
+  return newTrip
+}
+  function showNewTrip(e) {
+    const newTrip = getFormData(e);
+    const postPromise = postNewTrip(newTrip);
+    const newFetchPromise = fetchApiData('http://localhost:3001/api/v1/trips');
+    Promise.all([postPromise, newFetchPromise]).then(value => {
+      tripsRepo = new TripsRepository(value[1].trips);
+      showPostedTrip(value[0].newTrip)
+    });
+  }
+
+  function formatPostedTrip (trip) {
+      const destination = destinationRepo.getDestinationById(trip.destinationID);
+      const personPeople = formatNumTravelersGrammar(trip.travelers)
```

Comment:
Love the attention to detail with changing the wording of person/people based on how many travelers there are!